helpers.test.js

```diff
@@ -205,3 +206,152 @@ describe("Unique random functions", () => {
     });
   });
 });
+
+describe("readActionArguments", () => {
+  const { readActionArguments } = helpers;
+
+  const useNoAccountConfig = () => process.argv.splice(2, 1, path.resolve(__dirname, "__mocks__", "no-account", "app.js"));
```

Comment:
1. `__mocks__` is an internal jest feature used to automatically load manual mocks, if you'd like to define a general purpose mocks, just create a regular `mocks` directory.

2. Doesn't lint complain about the line length?

3. This is a side effect. Currently it doesn't cause any trouble, but if we forget about this line and expand the library at some point it may. Let's separate the config related functions to a separate file and mock it. We don't export it, so it wont break any contract.